pkg/controller/supportbundlecollection/store/supportbundlecollection.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2022 Antrea Authors
```

Comment:
It should be "support_bundle_collection.go". But could we just name it "collection.go"?

Reply:
Renamed it as collection.go